proposals/PluginsAudit2015/audit.md

```diff
@@ -0,0 +1,223 @@
+
+## Globalization
+* 1 Month Downloads: 5108
+* Open Issues: 12
+* Open Pull Requests: 0
+* Deprecate async calls
+* query all these values at startup and provide a synchronous API to get these values
+* Maybe it would be better to have JS libraries handle this?
+* Recommendation: Keep. Deprecate async calls and make plugin sync.
+
+## In-App-Browser
+* 1 Month Downloads: 17904
+* Open Issues: 116
+* Open Pull Requests: 36
+* API needs a refactor
+* Cordova Webview based inappbrowser as a replacement would have big security issues if we opened up third party links in it. So we shouldn’t do it.
+* Carlos suggested new inappbrowser based on safariview. He can elaborate more.
+* encourage people to use it through apis instead of window.open shims
+* demo of oauth workflow + demo of closing inappbrowser via js (url change)
+* http://ngcordova.com/docs/plugins/oauth/
+* get rid of insertcss & executescript. Use fileapi instead
+* Recommendation: Keep. Needs a refactor. Remove insertcss & executescript. Encourage use through apis instead of window.open. Make demos doing oAuth. Maybe phonegap-plugin-oauth.
+
+## Media 
+* 1 Month Downloads: 6396
+* Open Issues: 68
+* Open Pull Requests: 19
+* Too many quirks between platforms
+* Maybe implement web audio api spec instead as a plugin
+* web audio supported in iOS 8.4+ and Android 5+
+* create web audio plugin for Android 4 to 4.4
+* alternative: https://www.npmjs.com/package/cordova-plugin-nativeaudio
+* Recommendation: Deprecate! Recommend using web audio api instead and look into creating a web audio api plugin to polyfill support for android 4 to 4.4. 
+
+## Media-Capture
+* 1 Month Downloads: 5092
+* Open Issues: 37
+* Open Pull Requests: 9
+* Spec: http://www.w3.org/TR/mediacapture-streams/ (Last call draft)
+* Needs a api refactor
+* Issues with localization/globalization for custom UI
+* Alternatives?
+* https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Browser_compatibility or https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia (deprecated)
+* WebAudio can handle playing audio
+* camera plugin can handle video and pics
+* recording audio is missing. Options:
+* https://www.npmjs.com/package/cordova-plugin-audio-recorder-api
+* https://www.npmjs.com/package/cordova-plugin-audiorecorder
+* Recommendation: Keep for now, but look at alternatives and see if we can deprecate. If we do want to keep it long term, it will need a big refactor to catch up to spec
+
+## Network-Information
+* 1 Month Downloads: 13418
+* Open Issues: 14
+* Open Pull Requests: 0
+* Spec http://www.w3.org/TR/netinfo-api/ (discountinued)
+* Maybe this should be extended?
+* extra properties if cell data? 
+* Recommendation: Keep, investigate to see if extending functionality makes sense
+
+## Status-Bar
+* 1 Month Downloads: 33072
+* Open Issues: 27
+* Open Pull Requests: 3
+* no spec
+* pretty much required for iOS
+* Recommendation: Keep, add to hello world template
```

Comment:
Can this be handled without an explicit API by parsing meta tags for `theme-color`?

If the theme-color is solid, show the status bar in that colour. If the theme-color has alpha, tint the status bar that colour and overlay it on the webview.